smartcontractkit/smart-contract-db
# feat: adds deploy page

# Overview
Adds deploy page

# Screenshots for Mobile and Desktop views (if applicable)
![image](https://user-images.githubusercontent.com/20880360/137027696-ab4fbc74-b774-4eb4-92c3-3e1d413da7fe.png)


### Manual Testing

Write the steps required for how might test this. Example:

1. Run `npm run dev`
2. In browser, goto `localhost:3000/`
3. Navigate to deploy and verify items
4. Get results :tada:
